Automattic/wp-calypso
# composite checkout: RTL support

#### Changes proposed in this Pull Request

This adds RTL styles to `@automattic/composite-checkout` and its Calypso integration layer. There's discussion about using a [stylis plugin](https://github.com/emotion-js/emotion/issues/793) to automate RTL conversion for emotion styled components, but this PR just does it manually.

<img width="1256" alt="Screen Shot 2020-06-24 at 9 29 50 AM" src="https://user-images.githubusercontent.com/942359/85565582-478d0900-b5fd-11ea-98b7-5841d2bd904f.png">

#### Testing instructions

* If you're not already using local calypso in an RTL mode: in your `config/development.json`, change the line `"rtl": false,` to `"rtl": true,` and set your language in `me/account` to an RTL language such as Hebrew or Arabic. See also https://github.com/Automattic/wp-calypso/blob/master/docs/rtl.md
* Visit Checkout with a number of items in your cart (bundled, not bundled, renewals, etc.) using `?flags=composite-checkout-force`
* Click around the interface and verify that RTL styles are applied. Note spacing/padding between elements, icon alignment, text alignment, one-sided borders, etc. Add/remove coupons, remove items (to check the modal).
* Repeat using mobile styles
